app/controllers/article_controller.rb

```diff
@@ -70,30 +70,43 @@ def eds_init
     config.show.abstract = 'eds_abstract'
     config.show.plink = 'eds_plink'
     config.show.route = { controller: 'article' }
-
-    # solr fields to be displayed in the show (single result) view
-    #   The ordering of the field names is the order of the display
-    config.add_show_field 'eds_title', label: 'Title'
-    config.add_show_field 'eds_composed_title', label: 'Composed Title'
-    config.add_show_field 'eds_other_titles', label: 'Other Titles'
-    config.add_show_field 'eds_languages', label: 'Language'
-    config.add_show_field 'eds_physical_description', label: 'Physical Description'
-    config.add_show_field 'eds_source_title', label: 'Journal'
-    config.add_show_field 'eds_database_name', label: 'Database'
-    config.add_show_field 'eds_authors', label: 'Author'
-    config.add_show_field 'eds_author_affiliations', label: 'Author Affiliations'
-    config.add_show_field 'eds_author_supplied_keywords', label: 'Author Supplied Keywords'
-    config.add_show_field 'eds_publication_type', label: 'Format'
-    config.add_show_field 'eds_document_type', label: 'Document Type'
-    config.add_show_field 'eds_publication_date', label: 'Publication Date'
-    config.add_show_field 'eds_publication_info', label: 'Published'
-    config.add_show_field 'eds_abstract', label: 'Abstract'
-    config.add_show_field 'eds_notes', label: 'Notes'
-    config.add_show_field 'eds_subjects', label: 'Subjects'
-    config.add_show_field 'eds_subjects_geographic', label: 'Geographic Subjects'
-    config.add_show_field 'eds_subjects_person', label: 'Person Subjects'
-    config.add_show_field 'eds_document_oclc', label: 'OCLC'
-    config.add_show_field 'eds_document_doi', label: 'DOI', helper_method: :doi_link
+    config.show.sections = {
+      'Summary' => {
+        eds_authors: 'Authors',
+        eds_author_affiliations: 'Author Affiliations',
+        eds_source_title: 'Journal',
+        eds_publication_date: 'Publication Date',
+        eds_languages: 'Language'
+      },
+      'Abstract' => {
+        eds_abstract: 'Abstract',
+        eds_notes: 'Notes'
+      },
+      'Subjects' => {
+        eds_subjects: 'Subjects',
+        eds_subjects_geographic: 'Geography',
+        eds_subjects_person: 'Person Subjects',
+        eds_author_supplied_keywords: 'Author Supplied Keywords'
+      },
+      'Details' => {
+        eds_publication_type: 'Format',
+        eds_document_doi: 'DOI',
+        eds_database_name: 'Database',
+        eds_composed_title: 'Composed Title',
+        eds_volume: 'Volume',
+        eds_series: 'Series',
+        eds_issue: 'Issue',
+        eds_page_start: 'Page Start',
+        eds_page_count: 'Page Count',
+        eds_isbns: 'ISBN',
+        eds_issns: 'ISSN',
+        eds_publication_info: 'Published',
+        eds_document_oclc: 'OCLC',
+        eds_document_type: 'Document Type',
+        eds_other_titles: 'Other Titles',
+        eds_physical_description: 'Physical Description'
```

Comment:
One general concern I have about this approach is that we no longer have the ability to use configurations that we might want/need down the road (e.g. the `helper_method` that was removed, linking to searches, etc).

It's tough to say now if we'll need to break completely out of the BL configurations in order to derive our customized display, but as-is, this might make it harder to do something along the lines of link authors or subjects to searches (unless I'm missing something about this approach that would allow that).

One alternate approach might be to still configure all the show fields via `add_show_field` (allowing us to configure `heper_methods`, etc) but loop through a sections configuration in the UI can call `doc_presenter.field_value('field_name')` to continue to take advantage of the BL rendering pipeline.  This will do things like handle the `render_multivalued` that we've added to the presenter, and we can configure a helper to deal w/ the fields that we know to contain html (which is currently being handled by the `render_text_from_html` presenter method).

Happy to chat more about potential alternate approaches.